examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -133,14 +148,15 @@
         render=False,
         color_by_speed=False,
         restart_instance=True,
-        use_ballistic=True
+        use_ballistic=True,
+        disable_collisions=True
     ),
 
     # environment related parameters (see flow.core.params.EnvParams)
     env=EnvParams(
         horizon=HORIZON,
         sims_per_step=1,
-        warmup_steps=0,
+        warmup_steps=800,
```

Comment:
I chose this number as the amount of time needed before congestion sets in. This seems like something the wave calibration team should figure out.

Reply:
We don't actually want this then. We want to avoid congestion ever setting in.

Reply:
will also keep this separate